package.json

```diff
@@ -36,8 +36,8 @@
     "@nestjs/serve-static": "^4.0.1",
     "@nestjs/terminus": "^9.2.2",
     "@nestjs/typeorm": "^10.0.0",
-    "@nivo/calendar": "^0.84.0",
-    "@nivo/core": "^0.84.0",
+    "@nivo/core": "^0.87.0",
+    "@nivo/line": "^0.87.0",
```

Comment:
let's remove it from this package.json and move it to twenty-front/package.json (we are relocating all new dependencies to the packages package.json)